oak-search-elastic/src/test/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticTestServer.java

```diff
@@ -141,4 +142,20 @@ private void checkIfDockerClientAvailable() {
         assumeNotNull(client);
     }
 
+    /**
+     * Launches an Elasticsearch Test Server to re-use among several tests.
+     * @param args
+     */
+    public static void main(String[] args){
+        ElasticsearchContainer esContainer = ElasticTestServer.getESTestServer();
+        try {
+            System.out.println("Docker container with Elasticsearch launched at \""+esContainer.getHttpHostAddress()+
+                "\". Please PRESS ENTER to stop it...");
+            System.in.read();
+            esContainer.stop();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
since you are just wrapping the exception in a RuntimeException, you could remove the try-catch block and add the throws in the method declaration